### ISSUE TITLE
Support Forum in "Processes"

### ISSUE THREAD

**Issue by Tettsch** [decidim/metadecidim#97]
Hello everyone, 
i was just wondering whether it is intended that is not possible to post new questions in the [Support Forum in the meta decidim  ](https://meta.decidim.org/processes/supportforum/f/705/) or does it work and it is just me who does not find a possibility to ask a question? If so, is there another space and a better way to ask general questions?

![image](https://user-images.githubusercontent.com/1160706/152159901-a377eef2-0f1d-4c02-8fbe-00adb6b6f500.png)

Best regards
Torben

**Comment by ahukkanen** [comment]
Ping @andreslucena @carolromero, is this intentional?

What is the current channel where people can ask questions about using Decidim?

This particular place is linked to e.g. from:
https://decidim.org/faqs/

It also shows up on the first page of online search when searching with "decidim support".

**Comment by carolromero** [comment]
Hi, yes, this is intentional. We're just reorganizing both [MetaDecidim](https://meta.decidim.org/) and [Matrix](https://app.element.io/#/room/#decidim:matrix.org) in order to have it ready for the Decidim Fest. I'll post an update here in the next few days.

**Comment by carolromero** [comment]
Hi @Tettsch, as promised, here's the update: to answer doubts or questions about the use and administration of the platform we've created a room in Matrix [1]. We have also reorganized the navigation and spaces of Metadecidim [2] to facilitate onboarding for new members. Let us know your impression if you feel like it, feedback is welcome!

[1] https://matrix.to/#/#Support-admins:matrix.org
[2] https://meta.decidim.org/

@andreslucena we should also change the link that Antti mentioned above.
